ff/serving/serving.go

```diff
@@ -54,40 +57,50 @@ func NewTrainingDataServer(logger *zap.SugaredLogger) (*TrainingDataServer, erro
 func (serv *TrainingDataServer) TrainingData(req *pb.TrainingDataRequest, stream pb.OfflineServing_TrainingDataServer) error {
 	id := req.GetId()
 	name, version := id.GetName(), id.GetVersion()
+	featureObserver := prom_metrics.BeginObservingFeatureServe(name+" "+version, "training data")
```

Comment:
run go fmt

Reply:
what happens if name has a space in it or if version has a space in it. Also why make up a string called "training data", not good for to force users to memorize a fixed set of strings that metrics actually uses on the frontend, Try using a set of constants